ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutor.java

```diff
@@ -198,9 +198,9 @@ public void buildAndExecuteQuery(
           );
         }
 
-        topologyTestDriverContainer.getTopologyTestDriver()
-            .producedTopicNames()
-            .forEach(topicInfoCache::get);
+        //        topologyTestDriverContainer.getTopologyTestDriver()
```

Comment:
As discussed offline, we should not check in this PR will this commented out. We can place in a hack to bypass this for FK joins only and clean it up later in order to merge this PR sooner, but we cannot disable the check for existing tests as this is an important piece of coverage.